src/Cms/Blueprint.php

```diff
@@ -656,7 +663,7 @@ protected function normalizeOptions(
 			}
 		}
 
-		return array_merge($defaults, $options);
+		return [...$defaults, ...$options];
```

Comment:
Where I could see the argument is lines like this. Here the unpacking syntax doesn't add more clarity over `array_merge`. They are pretty much the same. But I think there the argument would be to just change it for consistency.